### DISCUSSION TITLE
hugo-redirect-landing-page/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#100] [Giscus]
# hugo-redirect-landing-page/

前言这两天看到「秦大叔」的博客文章《网站重启》提到，因为博客评论中别人留下的域名过期被黄网注册链接到了黄网，从而导致因涉黄问题而喝茶。可见在

https://www.eallion.com/hugo-redirect-landing-page/

**Comment by koobai** [comment]
背景图实现方式改成了你这样，直接写在样式里，能省不少代码～

**Reply by eallion** [reply]
是吧~ 这样还可以减少一次网络 requests，低碳环保。
